float64 hot_lati
float64 hot_longi
float32 hot_alti

float32 hot_speed
float32 hot_radius
int16 hot_clockwise
